Hmm, this was originally added in https://github.com/immich-app/immich/commit/43da8c2a7230e9347af0b14a530a691288673892#diff-da00458cdaeaea2314cb0e0101c85130593048072ada62de01727958c5d6ca37R61. And noted during a review of a PR that reduced the Docker image size (https://github.com/immich-app/immich/pull/1523#pullrequestreview-1282246519). Unfortunately, it does break `npm i` on Windows with the following error:

```
"C:\Program Files\nodejs\npm.cmd" install
npm ERR! code EBADPLATFORM
npm ERR! notsup Unsupported platform for exiftool-vendored.pl@12.78.0: wanted {"os":"!win32"} (current: {"os":"win32"})
npm ERR! notsup Valid os:  !win32
npm ERR! notsup Actual os: win32

npm ERR! A complete log of this run can be found in: C:\Users\User1\AppData\Local\npm-cache\_logs\2024-04-08T09_30_18_297Z-debug-0.log

Process finished with exit code 1
```

--- a/server/Dockerfile
+++ b/server/Dockerfile
@@ -5,7 +5,7 @@ RUN apt-get install --no-install-recommends -yqq tini
 WORKDIR /usr/src/app
 COPY server/package.json server/package-lock.json ./
 RUN npm ci && \
-    # sharp-linux-x64 and sharp-linux-arm64 are the only ones we need
+    # exiftool-vendored.pl, sharp-linux-x64 and sharp-linux-arm64 are the only ones we need
     # they're marked as optional dependencies, so we need to copy them manually after pruning
     rm -rf node_modules/@img/sharp-libvips* && \
     rm -rf node_modules/@img/sharp-linuxmusl-x64
@@ -22,6 +22,7 @@ FROM dev AS prod
 RUN npm run build
 RUN npm prune --omit=dev --omit=optional
 COPY --from=dev /usr/src/app/node_modules/@img ./node_modules/@img
+COPY --from=dev /usr/src/app/node_modules/exiftool-vendored.pl ./node_modules/exiftool-vendored.pl
 
 # web build
 FROM node:iron-alpine3.18@sha256:fa5d3cf51725bd42d32e67917623038539dbe720dab082f590785c001eb4dfef as web

